client/components/edit/create/ParentControls.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from "react";
+
+interface ParentControlProps {
+    allowChangeParentPid: boolean;
+    parent: string;
+    handleParentChange: React.ChangeEventHandler<HTMLInputElement> | undefined;
+    noParent: boolean;
+}
+
+const ParentControl = ({ allowChangeParentPid, parent, handleParentChange, noParent }: ParentControlProps) => {
```

Comment:
Like StateControl, this may need to be reused in future in an editing rather than creation mode, so maybe it should go somewhere else. It will also eventually need to be improved to incorporate the ability to pick the parent PID out of a tree as an alternative to hand-entering it (though that's not top priority right now).

Reply:
Made change.